content/docs/user-guide/external-dependencies.md

```diff
@@ -1,22 +1,23 @@
 # External Dependencies
 
 There are cases when data is so large, or its processing is organized in a way
-such that you would like to avoid moving it out of its external/remote location.
-For example from a network attached storage (NAS), processing data on HDFS,
-running [Dask](https://dask.org/) via SSH, or having a script that streams data
-from S3 to process it.
+such that its preferable to avoid moving it from its original location, even if
+it's external or remote to the project. For example: data on a network attached
```

Comment:
`even if it's external or remote to the project` - no need for this. It complicates it.

Reply:
Removed in 8a6892dd.